panel/src/components/Dialogs/UploadDialog.vue

```diff
@@ -28,6 +28,11 @@
 								:src="file.url"
 								back="pattern"
 							/>
+							<k-image-frame
+								v-else-if="file.type.match('svg')"
```

Comment:
Would this also match a file whose extension only *contains* `svg` (e.g. `svgz` or `abcdesvgabcde`)? Same for the pixel formats above.

Reply:
Good point. I think we need to differentiate between upload and upload replace. The latter matches on the url. There it is on the extension. This one is on the mime type. I think both could have problems as you point out.

Reply:
Probably best to switch to using mime types for both dialogs. And then look for exact matches on the full mime type strings.

Reply:
```html
<k-image-frame
	v-if="
		[
			'image/jpeg',
			'image/jpg',
			'image/gif',
			'image/png',
			'image/webp',
			'image/avif'
		].includes(file.type)
	"
	:cover="true"
	:src="file.url"
	back="pattern"
/>
<k-image-frame
	v-else-if="file.type === 'image/svg+xml'"
	:src="file.url"
	back="pattern"
/>
```

Reply:
https://github.com/getkirby/kirby/pull/5461/commits/8760ce4236c68595d3f418a96c03a00d8c09974e